proposals/description/tip_118.ts

```diff
@@ -308,16 +308,105 @@ const tip_118: TemplatedProposalDescription = {
       the LendingPoolAddressesProvider.
       `
+    },
+
+    //// agEUR Redemption
+    {
+      target: 'angleDelegatorPCVDeposit',
+      values: '0',
+      method: 'claimGaugeRewards(address)',
+      arguments: (addresses) => [
+        addresses.angleGaugeUniswapV2FeiAgEur // gauge
+      ],
+      description: 'Claim all unclaimed ANGLE rewards'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.angleDelegatorPCVDeposit, // pcvDeposit
+        addresses.angle, // token
+        addresses.tribalCouncilSafe, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: 'Migrate all ANGLE to TC Multisig where it can be sold'
+    },
+    {
+      target: 'angleDelegatorPCVDeposit',
+      values: '0',
+      method: 'unstakeFromGauge(address,uint256)',
+      arguments: (addresses) => [
+        addresses.angleAgEurFeiPool, // token
+        '9295203165522303785936702' // all LP tokens
+      ],
+      description: 'Unstake all agEUR/FEI Uniswap tokens from gauge'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.angleDelegatorPCVDeposit, // pcvDeposit
+        addresses.angleAgEurFeiPool, // token
+        addresses.agEurUniswapPCVDeposit, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: 'Move all agEUR/FEI Uniswap tokens to PCV deposit'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatio(address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.agEurUniswapPCVDeposit, // pcvDeposit
+        addresses.agEurUniswapPCVDeposit, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: 'Remove all agEUR/FEI Uniswap liquidity & burn FEI'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.agEurUniswapPCVDeposit, // pcvDeposit
+        addresses.agEUR, // token
+        addresses.angleEuroRedeemer, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: 'Move all agEUR to redeemer'
+    },
+    {
+      target: 'angleEuroRedeemer',
+      values: '0',
+      method: 'redeem()',
+      arguments: (addresses) => [],
+      description: 'Redeem all agEUR for FEI and DAI and send to DAI PSM'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'removeDeposits(address[])',
+      arguments: (addresses) => [[addresses.agEurUniswapPCVDeposit, addresses.uniswapLensAgEurUniswapGauge]],
+      description: 'Remove agEUR addresses from CR oracle'
+    },
+    {
+      target: 'pcvGuardian',
```

Comment:
This should be `pcvGuardianNew`. I introduced this anti-pattern, should update to use a versioning naming scheme like `pcvGuardianV2`, `pcvGuardianV1` etc. but that can be part of a different PR